### ISSUE TITLE
Filter Submissions/OData by submission date

### ISSUE THREAD

**Issue by issa-tseng** [getodk/central#23]
And possibly submitter or other metadata-level attributes. See original forum post [here](https://forum.opendatakit.org/t/odk-central-v0-2-beta/15005/4).

**Comment by ggalmazor** [comment]
Related to this issue: while working on Briefcase v.16 we found that we can't support using a date range filter when exporting a form because it relies on Aggregate adding a submission date to the submission contents.

This won't prevent users from exporting *all* submissions, but we know there are users with large forms that need this to streamline their data management scripts by exporting fewer submissions each time.

**Comment by issa-tseng** [comment]
to clarify, you’re just providing historical context about aggregate’s limitations, yeah? you’re not saying you ran into a problem that will prevent central from providing this feature?

**Comment by ggalmazor** [comment]
Ugh, I realize that my comment was super hard to follow :/ In my defence I'll say that I think I hadn't had my first coffee yet :P

What I wanted to add is that we found that Central **does not** add a submission date to the submissions, which makes it impossible for Briefcase users to export them with a date range filter.

Of course, correct me if I'm wrong, please ;)

Aggregate does add submission date metadata to submissions, which was used by Briefcase to implement the date range filter for exports.

**Comment by issa-tseng** [comment]
central does not inject a submission date into the submission itself, but it does know that date, and it provides it on the REST API, on the OData export under `__system` metadata, and in CSVZIP export as an additional column at the end.
